### PR TITLE
Consolidate language scripts

### PR DESCRIPTION
**Pull request by scottilee**
Consolidates the separate language bash files into one.

### CODE REVIEW

File: openapi/client.sh [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    OPENAPI_GENERATOR_COMMIT="${OPENAPI_GENERATOR_COMMIT:-v3.3.4}"
+    CLEANUP_DIRS=(pkg)
+    ;;
+
+  haskell)
+    OPENAPI_GENERATOR_COMMIT="${OPENAPI_GENERATOR_COMMIT:-a979fd8e13c86431831b0c769ba7b484e744afa5}"
+    CLEANUP_DIRS=(lib/Kubernetes/OpenAPI/API)
+    ;;
+
+  java)
```
Comment (yue9944882) [inline openapi/client.sh]:
```suggestion
  java)
    OPENAPI_GENERATOR_COMMIT="${OPENAPI_GENERATOR_COMMIT:-v4.2.0}" \
```

java generation has a strict bound on the generator version

**Comment by tomplus** [comment]
To be honest I prefer separate files instead of one big file with many case/if inside. Duplicated code is not so big, scripts are tailored to the specific library and it looks clear for me.

**Comment by brendandburns** [comment]
I sort of agree with @tomplus 

Maybe an alternative would be to extract common functions into a `util.sh` but retain the `java.sh` etc as entry-points?

Also, this is going to break a bunch of the docs/workflows in the actual client libraries repos (e.g. https://github.com/kubernetes-client/javascript) so I think we need to retain the language specific scripts to make sure those workflows don't break.

**Comment by scottilee** [comment]
Hey everyone, thanks for the feedback! I will look into creating a util.sh script instead.

**Pull request closed without merging**